legal-api/src/legal_api/services/filings/validations/dissolution.py

```diff
@@ -167,7 +167,8 @@ def validate_parties_address(filing_json, legal_type) -> Optional[list]:
 def _is_dissolution_party_role(roles: list) -> bool:
     return any(role.get('roleType', '').lower() in
                [PartyRole.RoleTypes.CUSTODIAN.value,
-                PartyRole.RoleTypes.LIQUIDATOR.value] for role in roles)
+                PartyRole.RoleTypes.LIQUIDATOR.value,
+                PartyRole.RoleTypes.PROPRIETOR.value] for role in roles)
```

Comment:
this will need to include `PartyRole.RoleTypes.PARTNER` as well.

Reply:
@argush3 I had a discussion with @ritvick. According to the requirement this validation may not be required. He will update this PR.

Reply:
oh interesting. ok, sounds good.

Reply:
I have updated the PR. Can you please review @vysakh-menon-aot